frontend/spec/features/products_spec.rb

```diff
@@ -122,9 +118,9 @@
 
   it 'is able to search for a product' do
     fill_in 'keywords', with: 'shirt'
-    click_button 'Search'
+    first('input[type=submit]').click
```

Comment:
`fill_in 'keywords', with: 'shirt\n'` should both enter the text and "press" Enter

Reply:
Maybe `"shirt\n"` should be in double quotes to make it work

Reply:
@KirillKayumov please fix this

Reply:
Unfortunately, it doesn't work. Probably it works only when a JS driver is enabled. I can't enable it for this scenario either, so I'm going to leave it as is.